src/Core/tests/DeviceTests/Stubs/EntryStub.cs

```diff
@@ -13,5 +13,15 @@ public partial class EntryStub : StubBase, IEntry
 		public string Placeholder { get; set; }
 
 		public bool IsTextPredictionEnabled { get; set; }
+
+		public Keyboard Keyboard { get; set; }
+
+		public bool IsSpellCheckEnabled { get; set; }
+
+		public int MaxLength { get; set; }
+
+		public Color PlaceholderColor { get; set; }
+
+		public bool IsReadOnly { get; set; }
```

Comment:
Hmmmm. I think we need to remove these for now? @rmarinho @PureWeen

We can always add it back. Not sure if we are going to add all these props. @hartez did you have issues for each, or are there any that we are not going to do?